Tests/FunctionalTests/NetworkResponseHandlerFunctionalTests.swift

```diff
@@ -43,8 +43,8 @@ class NetworkResponseHandlerFunctionalTests: FunctionalTestBase {
     func testProcessResponseOnError_WhenGenericJsonError_dispatchesEvent() {
         setExpectationEvent(type: FunctionalTestConst.EventType.EDGE, source: FunctionalTestConst.EventSource.ERROR_RESPONSE_CONTENT, expectedCount: 1)
         let jsonError = "{\n" +
-            "\"namespace\": \"global\",\n" +
-            "\"message\": \"Request to Data platform failed with an unknown exception\"" +
+            "\"type\": \"global\",\n" +
```

Comment:
Let's use real examples like `"type":"https://ns.adobe.com/aep/errors/EXEG-0201-503"` instead of global. Applies for all occurrences of global.